### ISSUE TITLE
Provision new production RDS database in AWS

### ISSUE THREAD

**Issue by sfisher** [CDL-Dryad/dryad-product-roadmap#319]
We need a dryad production database to write migration data to.  This will be a task for ITS and Marisa and Jim are working with them to provision it.

The database should be accessible from our current stage instance `uc3-dryadui-stg-2c.cdlib.org` .

**Comment by sfisher** [comment]
We will do it on stage database server and then export it to production.

**Comment by marisastrong** [comment]
We will be creating a new RDS Stage instance to house the migration db.  This will keep us from accidentally writing content to the other stage db.

**Comment by marisastrong** [comment]
I requested a new RDS Production instance.  **rds-uc3-dryad-prd.**  Reasons being: At some point we should be using an RDS instance named Dryad for naming consistency.  Also since this is a new instance, we can request that access be allowed from the stage environment (this is not uncommon).  And writing to a separate production instance, will prevent us from accidentally writing content to stage db.  We'll see if IAS grants our request, if not, we'll use the current Dash2 RDS stage instance.  
I'll update the ticket when we know more.  
I requested Need by date of 5/31.
